src/game/common/bitflags.h

```diff
@@ -242,77 +439,89 @@ template<int bits> void BitFlags<bits>::Parse(INI *ini, Utf8String *string)
         string->Clear();
     }
 
-    const char *token = ini->Get_Next_Token_Or_Null();
-    bool adjust = false;
     bool set = false;
```

Comment:
These names aren't great but i'm really struggling to think of better names. Set represents being in the no operator state and adjust represents being in the operator state. I think having some comment on this would make it more clear as at first glance you would expect Set and Clear to be the two states.